rbc/omniscidb.py

```diff
@@ -862,6 +865,11 @@ def _make_udtf(self, caller, orig_sig, sig):
                 sizer_index = consumed_index + 1
                 sizer = _sizer
 
+            _input_id = annot.get('input_id', unspecified)
+            if _input_id is not unspecified:
+                annot["input_id"] = _input_id.replace("args", "")
```

Comment:
When we remove `args` part at this level, we basically restrict the syntax `a=foo<...>` to `foo == args` case only. The same note applies to https://github.com/omnisci/omniscidb-internal/pull/5719 (I will add the corresponding review comment there).
This will disallow introducing annotations like `input_id=foo<arg1, arg2>` if we ever would like to introduce `foo`.